.eslintrc

```diff
@@ -81,96 +95,97 @@
       0,
       "never"
     ],
-    "flowtype/type-id-match": [
-      0,
-      "^([A-Z][a-z0-9]+)+Type$"
-    ],
+    "flowtype/type-id-match": 0,
     "flowtype/union-intersection-spacing": [
-      1,
+      2,
       "always"
     ],
     "flowtype/use-flow-type": 0,
-    "flowtype/valid-syntax": 0,
 
     ////////// Possible Errors //////////
 
-    "comma-dangle": 1,         // disallow trailing commas in object literals
-    "no-cond-assign": 0,          // disallow assignment in conditional expressions
+    "comma-dangle": [             // disallow trailing commas in object literals
+      2,
+      "always-multiline"
+    ],
+    "no-cond-assign": 2,          // disallow assignment in conditional expressions
     "no-console": 1,              // disallow use of console (off by default in the node environment)
-    "no-constant-condition": 0,   // disallow use of constant expressions in conditions
-    "no-control-regex": 0,        // disallow control characters in regular expressions
+    "no-constant-condition": 1,   // disallow use of constant expressions in conditions
```

Comment:
I don't think this rule is useful as it catches `while(true) {}` statements which in mind are completely fine. Let's disable it for now.